From 5cf17ebc659c875aff3c49d2a59ce15f46167389 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Fri, 9 May 2025 10:19:37 -0300
Subject: [PATCH 50/56] hurd: Fix tst-stack2 test build on Hurd

It requires $(shared-thread-library).  Fixes 0c342594237.

Checked on a i686-gnu build.

(cherry picked from commit f66cb3c9ebcac80b3200c3aff0e3aed6111547ba)
---
 sysdeps/pthread/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sysdeps/pthread/Makefile b/sysdeps/pthread/Makefile
index 17bdd6f49d..7fcbc72bc0 100644
--- a/sysdeps/pthread/Makefile
+++ b/sysdeps/pthread/Makefile
@@ -543,6 +543,7 @@ $(objpfx)tst-create1: $(shared-thread-library)
 $(objpfx)tst-create1.out: $(objpfx)tst-create1mod.so
 
 $(objpfx)tst-stack2.out: $(objpfx)tst-stack2-mod.so
+$(objpfx)tst-stack2-mod.so: $(shared-thread-library)
 LDFLAGS-tst-stack2-mod.so = -Wl,-z,execstack
 ifeq ($(have-no-error-execstack),yes)
 LDFLAGS-tst-stack2-mod.so += -Wl,--no-error-execstack
-- 
2.49.0

